src/js/control-bar/picture-in-picture-toggle.js

```diff
@@ -46,6 +43,18 @@ class PictureInPictureToggle extends Button {
     return `vjs-picture-in-picture-control ${super.buildCSSClass()}`;
   }
 
+  /**
+   * Enables or disables button based on document.pictureInPictureEnabled property value
+   * or on value returned by player.disablePictureInPicture() method.
+   */
+  handlePiPEnabledChange() {
```

Comment:
What do you think of calling this `handlePictureInPictureEnabledChange` for consistency. I know it's really long, but it seems weird to only use `PiP` here when we use `PictureInPicture` everywhere else.

Reply:
Hi, I don't have a problem with that. I have just changed the method name to fit your expectations.